baton_capabilities.json

```diff
@@ -1,20 +1,60 @@
 {
-  "@type": "type.googleapis.com/c1.connector.v2.ConnectorCapabilities",
-  "resourceTypeCapabilities": [
+  "@type":  "type.googleapis.com/c1.connector.v2.ConnectorCapabilities",
+  "resourceTypeCapabilities":  [
     {
-      "resourceType": {
-        "id": "user",
-        "displayName": "User",
-        "traits": [
+      "resourceType":  {
+        "id":  "dataset",
+        "displayName":  "Dataset",
+        "traits":  [
+          "TRAIT_APP"
+        ],
+        "description":  "Dataset of Google BigQuery"
+      },
+      "capabilities":  [
+        "CAPABILITY_SYNC"
+      ]
+    },
+    {
+      "resourceType":  {
+        "id":  "role",
```

Comment:
needs project

Reply:
Maybe we don't want project yet because there are no entitlements

Reply:
Fixed